examples/small-study/src/background.js

```diff
@@ -0,0 +1,76 @@
+// Allow outside override for debug and testing using prefs
+const STUDYPREFS = {
+  variationPref: `shield.${addonWidgetId}.testingBranch`,
+  firstrunPref: `shield.${addonWidgetId}.firstrun`,
+};
+
+const STUDYCONFIG = {
+  // the study
+  weightedVariations: {},
+};
+
+// This is a Shield Study template.
+class BaseStudy {
+  constructor(variation) {
+    // also sets activeExperiment key
+    // makes telmemetry possible
+    browser.study.configure(variation, ...STUDYCONFIG);
```

Comment:
Does not validate the schema https://github.com/mozilla/shield-studies-addon-utils/blob/121-finalize-schema/webExtensionApis/study/src/schemas/schema.studySetup.json - is the suggestion to simplify the schema to match the current small-study or will small-study be updated to validate the schema?

Reply:
Some of both.  I just finally went through Config.jsm to see what we can drop.  There are some keys we don't need any more, and some that I would like to rename.  Configure was badly designed then, and we can do better.